localstack/services/lambda_/provider.py

```diff
@@ -1535,10 +1535,16 @@ def create_event_source_mapping(
         context: RequestContext,
         request: CreateEventSourceMappingRequest,
     ) -> EventSourceMappingConfiguration:
-        if "EventSourceArn" not in request:
+        service = None
+
+        if "SelfManagedEventSource" in request:
```

Comment:
here an req will look like:

```
{
  'FunctionName': '<fn>',
  'Topics': [
    'lambda.events'
  ],
  'SourceAccessConfigurations': [
    {
      'Type': 'BASIC_AUTH',
      'URI': '<secret>'
    }
  ],
  'SelfManagedEventSource': {
    'Endpoints': {
      'KafkaBootstrapServers': [
        '<host:port>'
      ]
    }
  },
  'SelfManagedKafkaEventSourceConfig': {
    'ConsumerGroupId': '<group-id>'
  }
}
```

Anything else to validate it as a kafka?

Reply:
That's a reasonable check 👍